### DISCUSSION TITLE
Option to disable the Highlight boxes from Albums

### DISCUSSION THREAD

**Discussion by Aswin8** [deckerst/aves#783] [Ideas]
Please add an Option to disable the bright  Highlight boxes that is present around the pics on the Albums.

It's looking too brighter than the actual content and the pictures.

Apart from that the app is flawless and fluid.

**Comment by deckerst** [comment]
You can disable highlights in Settings > Display (for the whole app, not only the albums). I don't intend to provide highlight settings for specific pages.

**Reply by Aswin8** [reply]
Thanks for replying but i meant the white boxes that appear in Album even after disabling the highlights in settings.

**Reply by deckerst** [reply]
Borders are meaningful and consistent in the app, showing it's a filter.
